File: src_simple/f90/perimetre.f90
! perimetre  -  11.03.2014

! ******************************************
! Calcul du périmètre d'un cercle
! ******************************************

program prog_perimetre
    
    implicit none

    ! parametres
    double precision, parameter :: pi = 3.141592653

    ! variables
    double precision :: rayon, perimetre

    ! lecture du rayon
    write(*, *) "Rayon = "
    read(*, *) rayon
    write(*, *) "Rayon = ", rayon
    
    ! calcule le perimetre
    perimetre = 2. * pi * rayon
    write(*, *) "Perimetre = ", perimetre

end program prog_perimetre

File: src_simple/python/perimetre.py
#!/usr/bin/env python
# -*- coding=utf-8 -*-

""" perimetre  -  11.03.2014 """

import math

def perimetre(rayon):
    """ Calcul du périmètre d'un cercle """
    return 2.0 * math.pi * rayon

if __name__ == "__main__":
    # lecture du rayon
    rayon = float(raw_input("entrer le rayon : "))
    print("rayon = {0}".format(rayon))

    # affichage du résultat
    print("périmètre = {0}".format(perimetre(rayon)))

